node.js_mongodb/how to use.txt

```diff
@@ -0,0 +1,29 @@
+The update comes with two additions to upload data into MongoDB: SingleUpload.js & BulkUpload.js
```

Comment:
This is great documentation! Thanks for adding this. Can you rename it to README.md?